src/Sylius/Bundle/AdminApiBundle/Resources/config/serializer/Model.Order.yml

```diff
@@ -6,6 +6,14 @@ Sylius\Component\Core\Model\Order:
             expose: true
             type: string
             groups: [Default, Detailed, DetailedCart]
+        paymentState:
+            expose: true
+            type: string
+            groups: [Default, Detailed, DetailedCart]
+        shippingState:
+            expose: true
+            type: string
+            groups: [Default, Detailed, DetailedCart]
```

Comment:
I wonder, should they be included in the `DetailedCart` group as well? When the order is not placed, they're always `'cart'`

Reply:
I haven't followed up on the usage of the `DetailedCart` group, I just copied the `checkoutState` because I thought they would be relevant the same places as the `checkoutState`